diff -aurN poldek-0.30.1.orig/conf/poldek.conf poldek-0.30.1/conf/poldek.conf
--- poldek-0.30.1.orig/conf/poldek.conf	2014-03-26 21:48:39.868726416 +0100
+++ poldek-0.30.1/conf/poldek.conf	2014-03-26 21:49:34.610857802 +0100
@@ -128,7 +128,7 @@
 #confirm removal = yes
 
 # Let the user select package among equivalents.
-#choose equivalents manually = yes
+choose equivalents manually = yes
 
 # Prevent package listed from being upgraded if they are already installed.
 # hold = kernel* foo*.i686 th-test:* *-smp-* th-ready:bar*.x86_64
